### ISSUE TITLE
no easy way to get by type when it returns multiple elements

### ISSUE THREAD

**Issue by DeanPack** [appium-userland/appium-flutter-driver#322]
I have a login screen with two text fields. one for an email and one for a password. In my flutter tests I enter this
await tester.enterText(find.byType(TextField).first, email);
trying to get .first from a appium flutter finder makes it return undefined.

**Comment by DeanPack** [comment]
on my app .first on the find by type returns the email field, and if I do finder[1] I get my password field.

**Comment by badri001** [comment]
Yes, Even i am facing difficulties while the finder returns multiple elements . in flutter driver i easy specify the first element by finder.first but here i am feeling fuzzy and stuck.
